llvm/tools/llvm-objcopy/ObjcopyOpts.td

```diff
@@ -234,11 +234,11 @@ defm update_section
 defm gap_fill
     : Eq<"gap-fill", "Fill the gaps between sections with <value> instead of zero. "
                      "<value> must be an unsigned 8-bit integer. "
-                     "This option is only supported for ELF inputs and binary outputs.">,
+                     "This option is only supported for the ELF input and binary output">,
```

Comment:
I don't think you want "the" before "ELF".

Reply:
Removed.